src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommand.cs

```diff
@@ -181,9 +181,17 @@ public void InstallWorkloads(IEnumerable<WorkloadId> workloadIds, bool skipManif
 
             InstallWorkloadsWithInstallRecord(workloadIds, _sdkFeatureBand, manifestsToUpdate, offlineCache);
 
-            if (_workloadInstaller.GetInstallationUnit().Equals(InstallationUnit.Packs))
+            try
+            {
+                if (_workloadInstaller.GetInstallationUnit().Equals(InstallationUnit.Packs))
+                {
+                    _workloadInstaller.GetPackInstaller().GarbageCollectInstalledWorkloadPacks();
```

Comment:
Is this the only place we call garbage collection?  Doesn't it happen on uninstall and update too?  Shouldn't we also be warning the same way anywhere it's called?